Makefile

```diff
@@ -0,0 +1,8 @@
+.PHONY: develop test
+
```

Comment:
I don't think `py.test` would work with the existing `runtests.py` code (but I may be wrong). `Makefile`s are used across a number of popular Django projects in this way (see [wagtail](https://github.com/torchbox/wagtail/blob/master/Makefile), [django-model-utils](https://github.com/carljm/django-model-utils/blob/master/Makefile), [django-activity-stream](https://github.com/justquick/django-activity-stream/blob/master/Makefile) etc.). What about them do you dislike?